recipes/leftbar/general_row/general_row.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <li
+    :class="leftbarGeneralRowClasses"
+    data-qa="leftbar-general-row"
+  >
```

Comment:
hmm yeah making it an `<a>` and then wrapping the `<a>` in a router-link doesn't really make sense since I believe the router link is an `<a>` itself.... Let me look a bit more into how router-link works, there might be some options to help with this.

Reply:
In DP 2.0 the structure is

```html
<li>
  <RouterLink>
    <div>
```

So maybe we just leave this as a div that would be wrapped by routerlink

Reply:
Updated to div so in the product you can wrap it with an anchor or router link https://github.com/dialpad/dialtone-vue/pull/769/commits/2ea42d1ddf06c03d9d0b9208136be33eab276b48